services/web-app/utils/mdx.js

```diff
@@ -94,3 +95,24 @@ export async function processMdxSource(mdxSource, url) {
     warnings
   }
 }
+
+export async function getProcessedMdxSource(params, url) {
```

Comment:
This function could benefit from some JSDoc explaining what it does, why you'd use it, and what happens for both the happy path and the error paths.

Reply:
Done!